.github/scripts/create-fork-pipeline-changes.js

```diff
@@ -0,0 +1,49 @@
+const saveFile = require('fs').writeFileSync;
+
+function packageChanges(rootPath, owner) {
+  const pkgJsonPath = `${rootPath}/package.json`
+  const json = require(pkgJsonPath);
+
+  if (!json.hasOwnProperty('repository')) {
+    json.repository = {};
+    json.repository['type'] = 'git'
+    json.repository['url'] = `https://github.com/${owner}/twilio-cli.git`
+  } else {
+    json.repository['url'] = `https://github.com/${owner}/twilio-cli.git`
+  }
+
+  if (json.hasOwnProperty('oclif')) {
+    if (!json.oclif.hasOwnProperty('macos')) {
+      json.oclif.macos = {};
+      json.oclif.macos['identifier'] = "com.twilio.cli"
+    } else {
+      delete json.oclif.macos.sign
```

Comment:
[Q] What will this delete exactly?

Reply:
If we don't remove sign from package.json then oclif-dev pack:macos will fail as it will try signing mac binary